main.tf

```diff
@@ -97,6 +114,41 @@ resource "azurerm_kubernetes_cluster" "aks" {
   }
 }
 
+resource "azurerm_kubernetes_cluster_node_pool" "example" {
```

Comment:
Resource needs to be renamed from "example"

Reply:
good catch